views/pages/contact.ejs

```diff
@@ -17,7 +17,7 @@ h1, p {
 }
 
 input {
-	border: 2px solid darkgrey;
+	border: 2px solid darkred;
```

Comment:
why darkred? grey fitted better with the theme, but please tell me otherwise what you think